ckanext/depositar_theme/templates/home/index/idx_header.html

```diff
@@ -0,0 +1,139 @@
+{% macro language_selector() %}
+  {% set current_lang = request.environ.CKAN_LANG %}
+  {% if current_lang != "en" %}
+    <li>
+      <a href="{% url_for h.current_url(), locale='en' %}">English</a>
+    </li>
+  {% endif %}
+  {% if current_lang != "zh_Hant_TW" %}
+    <li>
+      <a href="{% url_for h.current_url(), locale='zh_Hant_TW' %}">中文</a>
+    </li>
+  {% endif %}
+{% endmacro %}
```

Comment:
覆蓋必要 block 即可，其他可以使用 super()

```suggestion
{% extends 'header.html' %}
```